tests/tape/test_tape.py

```diff
@@ -1658,6 +1658,22 @@ def test_creating_scratch_tape(self):
         assert len(temp_tape.operations) == 1
         assert temp_tape.operations[0] == op1
 
+    @pytest.mark.parametrize("ret", [expval, var])
```

Comment:
Since this test depends on an error being raised by a specific transform, and we intend to fix that error, this might not make the best unittest for the behavior of `qml.QueuingManager.stop_recording`.

Maybe instead we could add a test to `test_queuing.py` like:
```
with qml.queuing.AnnotatedQueue() as q:
    with qml.QueuingManager.stop_recording():
        raise ValueError
```
and make sure that a `ValueError` is raised instead of an `IndexError`. No more dependence on the insert transform.

Reply:
much cleaner, updated! tbh I was just copying an existing test because this debug all happened so quickly :p